draft-ietf-rats-corim.md

```diff
@@ -2417,6 +2501,28 @@ If key verification succeeds for any _key_:
 
 Otherwise, do not add the `addition` ECT to the ACS.
 
+#### Processing Domain Membership {#sec-process-dm}
+
+This section assumes that each Domain Membership Triple has been transformed into an internal representation following the steps described in {{sec-ir-dm-trans}}, resulting in the representation specified in {{sec-ir-dm}}.
+
+##### First Pass {#sec-first-pass}
+Domain Membership ECTs (`cmtype`: `domain-member`) in the staging area are matched with ACS entries (of `cmtype`: `evidence`) using the following algorithm:
+
+* For every `domain` ECT entry (`cmtype`: `domain-member`):
+  * For each `i` in `members`, check that there is a corresponding ACS entry with a matching `environment` and `cmtype`:`evidence`
+  * If all `members` match a corresponding ACS entry, add the `domain` ECT to ACS
+  * If none of the `members` match, proceed to next `domain` ECT in the list.
+  * If there is a partial match, i.e. some `i's` in `members` has a matching ACS `environment` and `cmtype`:`evidence` add the `domain` ECT entry (`cmtype`: `domain-member`) to the ACS and proceed to ({{sec-second-pass}})
+
+##### Second Pass {#sec-second-pass}
```

Comment:
There is no first pass and second pass.
There are as many passes needed to converge - or to realise it won’t.

Reply:
I think we need more time than is available before the deadline to put this into shape.  I propose that we:
1. Remove the content from the “Processing Domain Membership” section;
2. Raise an issue regarding its completion;
3. Add the issue link here as a placeholder.

Reply:
What we need to describe is an iterative algorithm that tries to converge to a fixed point in terms of 3 fundamental pieces:

* initialisation,
* iterative step procedure,
* stop condition.

There should be no real difference between matching against evidence ECTs and domain ECTs. At each pass, we try to match as much as possible, add what matched, and start again until we can match no more.

Anyway, if others think it's worth merging this section as-is, I won't stand in the way. If so, please create an issue to track improvements to the processing algorithm description.

Reply:
Fully agree with @thomas-fossati suggestion, so we have addressed this in the subsequent commit simplifying the algorithm greatly and doing justice with it.

Andy and Dionna are on board with this and agree the text in principle.